src/coreclr/jit/emitarm64.cpp

```diff
@@ -4250,9 +4250,11 @@ void emitter::emitIns_Mov(
 
         case INS_sve_mov:
         {
-            if (isPredicateRegister(dstReg) && isPredicateRegister(srcReg))
+            // TODO-SVE: Remove check for insOptsNone() when predicate registers
+            //  are present.
```

Comment:
Do we know what's causing this new TODO?

Reply:
Since predicate registers are alias of vector registers, it would just match the first condition, regardless of what type of registers. We have similar TODO at other places that will be removed once we add proper predicate registers.

Reply:
Ah, I see. That's definitely unfortunate. I'd guess it will also become a real problem once we get into the instructions that move/convert between vector and mask registers.

Reply:
IIRC, we have TODO listed at various places to handle it and we have added special `insOpts` for them, which will be removed once predicate register support is added.